utils/data.py

```
@@ -0,0 +1,97 @@
+import json
+from typing import Any, List, Optional
```

Ruff (0.14.11)

`typing.List` is deprecated, use `list` instead

(UP035)